clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -635,10 +674,16 @@ object GarbageCollector {
       .toDF(schema.fieldNames: _*)
   }
 
-  private def getMetadataMarkLocation(markId: String, gcAddressesLocation: String) = {
+  private def getMetadataMarkLocation(markId: String, gcAddressesLocation: String): String = {
     s"$gcAddressesLocation/$markId.meta"
   }
 
+  private def getRunIDMarkerLocation(runID: String, storagePrefix: String): String = {
+    var prefix: String = storagePrefix.stripPrefix("/").stripSuffix("/")
```

Comment:
@itaiad200 is this the right way to do it nowadays?  We've had so much pain around storage namespaces...

Reply:
Still worried about this one, though!